nautobot/extras/tables.py

```diff
@@ -674,7 +674,7 @@ class Meta(BaseTable.Meta):
         default_columns = ("created", "grouping", "log_level", "log_object", "message")
         row_attrs = {
             "class": log_entry_color_css,
-            "data-name": lambda record: record.log_level,
+            "data-name": lambda record: f"{record.log_level.capitalize()} {record.message}",
```

Comment:
Wonder if `data-name` remains the best name for this attribute. Also I'm a bit opposed to this duplication of content in the page - could we update the filtering JS to actually look at the text in the table cells instead of needing to render the message twice for each row?

Reply:
I was thinking much the same. Will have a play.